### PR TITLE
Removes Certain Religious References

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Removes all Islamic references from the game
## Why It's Good For The Game
I am an Arab and Muslim and I dislike seeing Islam in the game. I get why it was added, and I thank the guy that added it. He just wanted to show some representation for a demographic absent from SS13. But I find it quite distasteful, especially with the fact that you can fart and do other weird stuff with Bibles (which can be reskinned as Qurans)
It's also pretty bad during Ramadan, which is an entire MONTH. The station is always named something like Jihad Platform XV or Mohamad Station 9000 with dudes making joking around. It's their right, I mean it's the internet and free speech and whatever, but I still feel uneasy playing while that's going on and I can't be the only one

I'm not removing it for political reasons I just find it very distasteful and think it shouldn't be in the game. I pray IRL, it feels wrong having sacred (to me and other Muslims) stuff in this wacky game.

I'm not sure if I caught all the references so please inform me if I've missed some.
## Changelog
:cl:
del: Removed certain religious references out of respect
/:cl:

### CODE REVIEW

**Comment by musulmanSS13** [comment]
> I don't think you can fart on TG. And station usually gets renamed in a few minutes so thats quite weak argument.

You can still do stuff like throw it in the supermatter, disposals, deep fry it, etc.
I know you can rename it but why even have it in the first place? Ramadan is not even a holiday like people think of holidays like Christmas. It's more of a religious act of devotion for a month

**Comment by Peliex** [comment]
This removes all mentions of Ramadan from the roundstart text.  I get your personal problems with people picking untasteful names but hiding the holiday from people is probably not the right way to go about it?

**Comment by thElasT13** [comment]
now this is epic

**Comment by musulmanSS13** [comment]
> This removes all mentions of Ramadan from the roundstart text. I get your personal problems with people picking untasteful names but hiding the holiday from people is probably not the right way to go about it?

Personally I would rather not have anything to do with Islam in the game even mention of Ramadan

I don't really think it's necessary for people to know it's Ramadan. It's not a festive-type celebration, it's just a kind of devotional thing you do personally. Ramadan is just fasting till evening and reciting Quran a bit more since its a holy month.

Like why does it have to even be in the funny spessman game? I just consider it a holy thing and would rather not see it in entertainment. I would equally not like it if World of Warcraft had a Ramadan event but of course thats not open source so I can't open a PR there

**Comment by Pinkufeck** [comment]
This also removes, the islamic calendar, the name Hussain and Ahmed, and some clothes which significances go over my head so i won't comment about it, but seems like a weird first PR, especially since the github account was made today solely for this change. 
Like i'm all for the removal of distateful shit but you're removing it as a whole. Everything. Pretty weird.

**Comment by musulmanSS13** [comment]
> This also removes, the islamic calendar, the name Hussain and Ahmed, and some clothes which significances go over my head so i won't comment about it, but seems like a weird first PR, especially since the github account was made today solely for this change. Like i'm all for the removal of distateful shit but you're removing it as a whole. Everything. Pretty weird.

Yes I would like to remove all Islamic stuff from the game:
Islamic calendar code is only used to calculate Ramadan
Hussain and Ahmed are specifically Muslim names. Ahmed is an alternative shortened form of Muhammad and Hussain was the Prophet PBUH grandson who was martyred
I did not remove the name Khan even though its a common surname for Muslims since we don't have a monopoly on that name, i.e. Gengis Khan
The hat removed is the taqiyah aka topi. Its like a larger version of Jewish kippah basically, imams and more religious people wear it

**Comment by TheVekter** [comment]
While I can respect that there's some cultural concerns with having Islam in the game, I hope you can understand why we might think this is in bad faith given that you made a Github account explicitly for this purpose and have seemingly straight removed every reference to the religion including ones that, to my knowledge, would not read as offensive to an average worshiper of it.

It's like coming in out of nowhere and removing the names John, Josh, James, and Peter from the name list as well as every reference to Christmas from the game.

**Comment by Pinkufeck** [comment]
let me try to understand the logic behind it you, why do you not want to see stuff that's part of religion in the game ? Is it related to a tenet of islam or is it purely personal ?

**Comment by notfrying1pans** [comment]
this is bait

**Comment by norsvenska** [comment]
> It's also pretty bad during Ramadan, which is an entire MONTH. The station is always named something like Jihad Platform XV or Mohamad Station 9000 with dudes making joking around.

Correct me if I'm wrong, but I'm pretty sure the special names only appear for the first and last 3 days of Ramadan, not the entire month.

**Comment by Pinkufeck** [comment]
I'm sorry if this comes out as a little rude but this PR that removes all references to islam and names related to islam/middle east feels like a huge "Ahah i am a troll i finally got islam out of the spess game".

**Comment by musulmanSS13** [comment]
@TheVekter @Pinkufeck
Yes I understand why you guys are concerned. Wallahi I'm not trolling, this is not in bad faith. I'm not an Islamophobe or bigoted guy
In fact I would argue that someone who hates Muslims would prefer that it remains in the game so they can subtley hate on them. Note: I am not implying that those who want this content to remain in game are hateful to Muslims

Reason I want it removed is that I consider it a very sacred thing and wish to not have this in a video game which is just entertainment and is multiplayer online.

To give an exaggerated example, its like if a Jewish person was removing a readable Torah from the ss13 library. I know it's not in, but trying to give an example. We consider it sacred so we don't want it mixed with entertainment

**Comment by AtomTheProphet** [comment]
honestly maybe we should take this a step further, lets get rid of the bible entirely and just give chaplains a blank book. This entire PR is a joke, especially considering that it's their first.

**Comment by norsvenska** [comment]
> honestly maybe we should take this a step further, lets get rid of the bible entirely and just give chaplains a blank book. This entire PR is a joke, especially considering that it's their first.

Let's just remove the chaplain entirely. The whole thing is offensive to religious people because there can only be one. Who cares about cultists? Those are just a myth made up by NT to cause division in the crew! Oh, and while we're at it, let's remove all of the holidays because the names that come with them may be seen as offensive. I'm sure this will have zero retaliation from the playerbase.

**Comment by AtomTheProphet** [comment]
> > honestly maybe we should take this a step further, lets get rid of the bible entirely and just give chaplains a blank book. This entire PR is a joke, especially considering that it's their first.
> 
> Let's just remove the chaplain entirely. The whole thing is offensive to religious people because there can only be one. Who cares about cultists? Those are just a myth made up by NT to cause division in the crew! Oh, and while we're at it, let's remove all of the holidays because the names that come with them may be seen as offensive. I'm sure this will have zero retaliation from the playerbase.

Absolutely! And don't even tell me about the supposed "heretics", what nonsense! And honestly, if we're removing all of the holidays we might as well also stop doing things like april fools since there's a minuscule chance that something made for it could be seen as offensive.

**Comment by Serxule** [comment]
a few things
1. nobody is going to be hating on muslims/islam ingame, let alone using the few items there are that are related to do so, if they are then thats ahelpable, not a code issue (nobody gives a shit about religion or politics in funny space game)
2. i agree with the station name thing
3. unfortunately, this isnt a safespace or hugbox, not everyone will agree with everything, if you choose to play a game that has a few islamic references (which arent remotely disrespectful mind you, yes you can have an opinion on what is/isnt but there's still a baseline and it isnt disrespectful), but removing something religious entirely as a whole could very likely come across as trying to censor said religion and be considered disrespectful

**Comment by DATA-xPUNGED** [comment]
kinda silly

File: code/game/objects/items/grenades/flashbang.dm [hunk 1]
```diff
@@ -34,7 +34,7 @@
 		living_mob.Knockdown(max(200/max(1, distance), 60))
 
 //Bang
-	if(!distance || loc == living_mob || loc == living_mob.loc) //Stop allahu akbarring rooms with this.
+	if(!distance || loc == living_mob || loc == living_mob.loc)
```
Comment (MrStonedOne) [inline code/game/objects/items/grenades/flashbang.dm:37]: this is the only change allowed to stay.

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]
No

File: code/game/objects/items/grenades/flashbang.dm (hunk 1 above)
Comment (Toastgoats) [reply code/game/objects/items/grenades/flashbang.dm:37]: I think some of the station prefixes are pretty questionable too, thoughts on those?

**Comment by Capsandi** [comment]
I think this pr was made in good faith but there's no way this is going through

File: code/modules/holiday/holidays.dm [hunk 2]
```diff
@@ -708,30 +708,6 @@
...
-	return ..(cal.dd, cal.mm, cal.yyyy, ddd)
-
-/datum/holiday/islamic/ramadan
-	name = "Start of Ramadan"
-	begin_month = 9
-	begin_day = 1
-	end_day = 3
-
-/datum/holiday/islamic/ramadan/getStationPrefix()
-	return pick("Haram","Halaal","Jihad","Muslim", "Al", "Mohammad", "Rashidun", "Umayyad", "Abbasid", "Abdul", "Fatimid", "Ayyubid", "Almohad", "Abu")
```
Comment (DATA-xPUNGED) [inline code/modules/holiday/holidays.dm:727]: I'm not very knowledgeable on these but some of them seems quite tasteless

Comment (norsvenska) [reply code/modules/holiday/holidays.dm:727]: Haram: forbidden
Halaal: permissible
Jihad: struggle against enemies of Islam
Muslim: follower of Islam
Al: the
Mohammad: Praiseworthy; prophet
Rashidun, Umayyad, Abbasid: historic Islamic caliphates
Abdul: servant to god
Fatimid, Ayyubid, Almohad: historic Islamic dynasties
Abu: father (of)

The only two that I see as potentially tasteless are Jihad and (to a much lesser extent) Haram, but I'm no expert.

**Comment by MrStonedOne** [comment]
> I think this pr was made in good faith ...

It wasn't. Stoking the flames of culture war and sanitized language is a common astroturfing goal and given this is a new account using a username that can not be linked to any game or discord identity I'd be more likely to believe they are a astroturfer than believe they acting in good faith.

Given that both are still reasonably possible there is no reason not to remove the distasteful comment from flash bangs.

File: code/modules/holiday/holidays.dm (hunk 2 above)
Comment (Peliex) [reply code/modules/holiday/holidays.dm:727]: The only thing I discovered reading this is that we are shameless to all people during their respective holiday except Christmas, which doesn't have funny station names.

**Comment by TheVekter** [comment]
> > I think this pr was made in good faith ...
> 
> It wasn't. Stoking the flames of culture war and sanitized language is a common astroturfing goal and given this is a new account using a username that can not be linked to any game or discord identity, and they removed even the most harmless of references (like the hat) I'd be more likely to believe they are a astroturfer than believe they are acting in good faith.
> 
> Given that both are still reasonably possible there is no reason not to remove the distasteful comment from flash bangs.

I will probably spin up a PR later tonight to remove that comment and possibly a few of the prefixes.

**Comment by JohnFulpWillard** [comment]
> I will probably spin up a PR later tonight to remove that comment and possibly a few of the prefixes.

Thank you, I'll close this PR then.

File: code/modules/holiday/holidays.dm (hunk 2 above)
Comment (DATA-xPUNGED) [reply code/modules/holiday/holidays.dm:727]: > The only thing I discovered reading this is that we are shameless to all people during their respective holiday except Christmas, which doesn't have funny station names.

this is blasphemy

**Comment by optimumtact** [comment]
I don't believe the OP is here in good faith so I've blocked them permanently

**Pull request closed without merging**